routes/api/contacts-router.js

```diff
@@ -0,0 +1,55 @@
+import express from "express";
+import Joi from "joi";
+
+import contactsService from "../../models/contacts.js";
+import { HttpError } from '../../helpers/index.js';
+import {isEmptyBody, isValidId } from '../../middlewars/index.js';
+
+
+
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required().messages({
+    "any.required": `"name" must be exist`,
+  }),
+  email: Joi.string().required().messages({
```

Comment:
schema should be in separated file. You should validate email and phone properly